frontend/src/components/Tournament.js

```diff
@@ -24,6 +25,51 @@ import {
 } from "../skeletons/Standings";
 import Breadcrumbs from "./Breadcrumbs";
 
+const TeamSeedingChange = props => {
+  const filteredByTeamID = Object.entries(
+    props.tournament.initial_seeding
+  ).filter(([_, teamID]) => teamID == props.teamID); // eslint-disable-line no-unused-vars
+
+  const [initialSeed, _] = filteredByTeamID[0]; // eslint-disable-line no-unused-vars
+
+  return (
+    <Switch>
+      <Match when={props.currentSeed < initialSeed}>
```

Comment:
Can the initialSeed / props.currentSeed be strings by any chance here? Be better to parseInt here as well I think

Reply:
done, can you check now ?